Modules/Loadable/Segmentations/Resources/UI/qSlicerSegmentationsModule.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>348</width>
-    <height>719</height>
+    <height>844</height>
```

Comment:
this is a bit big... maybe collapse sections in the .ui file would reduce this?

Reply:
I've reduced the size as much as possible in the designer. If it is still too large, I can collapse some sections.